xbmc/platform/linux/input/LibInputKeyboard.cpp

```diff
@@ -148,39 +149,75 @@ static const std::map<xkb_keysym_t, XBMCKey> xkbMap =
   { XKB_KEY_XF86AudioRandomPlay, XBMCK_SHUFFLE }
   // XBMCK_FASTFORWARD clashes with XBMCK_MEDIA_FASTFORWARD
 };
+
+static void xkbLogger(xkb_context* context,
+                      xkb_log_level priority,
+                      const char* format,
+                      va_list args)
+{
+  const std::string message = StringUtils::FormatV(format, args);
+  auto logLevel = LOGDEBUG;
+  switch (priority)
```

Comment:
I'd prefer a map over a switch/case but that's minor.

Reply:
done